lib/linguist/heuristics.yml

```diff
@@ -73,6 +73,10 @@ disambiguations:
     pattern: '(<^\s*; |End Function)'
   - language: BitBake
     pattern: '^\s*(# |include|require)\b'
+- extensions: ['.bst']
+  rules:
+  - languages: BuildStream
+    pattern: '^*.'
```

Comment:
We don't have any other language using the `.bst` file extension, so this rule is unnecessary.